### DISCUSSION TITLE
COMMUNICATION STRATEGY MEETING

### DISCUSSION THREAD

**Discussion by charlieikosi** [pinpointltdpng/ODFD-PROJECT#2] [Announcements]
We will have a meeting next week Wednesday (29/03/23) to discuss communication methods. This will be via whatsapp conference call or zoom. Any questions please pose in the comments.

Cheers!

**Comment by anapitalai** [comment]
All good, looking forward to the meeting.

**Reply by charlieikosi** [reply]
Thanks alois. I just updated the date to Wednesday as I have an event to attend on Monday evening.
